docs/user/adding-new-metrics.md

```diff
@@ -6,54 +6,61 @@ The format of that file is documented [here](https://mozilla.github.io/glean_par
 
-toolbar:
-  click:
-    type: event
-    description: |
-      Event to record toolbar clicks.
-    notification_emails:
-      - CHANGE-ME@example.com
-    bugs:
-      - 123456789
-    data_reviews:
-      - http://example.com/path/to/data-review
-    expires: 2019-06-01  # <-- Update to a date in the future
+If the value is a string, use a [string metric](metrics/string.html).  If you want to store multiple strings in a group, use a [string list metric](metrics/string_list.html).
 
-  double_click:
-    ...
+If you need to record a closely-related group of strings, or you don't know the things the strings are related to at build time, use a [labeled string metric](metrics/labled_strings.html).
 
-category2.subcategory:  # Categories can contain subcategories
-  metric:
-    ...
+For all of these simple value types, you should consider how the value changes in the application and how that relates to the ping it is being sent in. When the ping is submitted, it will contain the "last known" value for the metric, and then the value is reset according to its [lifetime](#What-is-the-lifetime-of-the-data).
 
-```
+If you need to know the current state of a value, such as a user preference, to provide context for other metrics, be aware that the lifetime of the value may not be the same as the lifetime of the ping it is in. You can address this either with a custom ping that matches the lifetime of the value, or by using a user lifetime metric so the value will persist across pings.
 
-The details of the metric parameters are described in [metric parameters](metric-parameters.md).
+### Are you counting things?
 
-The `metrics.yaml` file is used to generate code in the target language (e.g. Kotlin, Swift, ...) that becomes the public API to access your application's metrics.
+If you want to know how many times something happened, use a [counter metric](metrics/counter.html).  If you are counting a group of related things, or you don't know what all of the things to count are at build time, use a [labeled counter metric](metrics/labeled_counter.html).
+
+If you need to know when the things being counted happened relative to other things, use an [event](metrics/event.html).
+
+### Are you measuring time?
+
+If you need to record an absolute time, use a [datetime metric](metrics/datetime.html). Datetimes are recorded in the user's local time, along with a timezone offset from UTC.
+
+If you need to record how long something takes you have a couple of options:
+
+If you need to measure the total time spent doing a particular task, look to the [timespan metric](metric/timespan.html). Note that it can only be used on a single thread.
 
-## Recommendations for defining new metrics
+If you need to measure the relative occurrences of many timings, use a [timing distribution](metric/timing_distribution.html). It builds a histogram of timing measurements, and is safe to use from multiple threads.
```

Comment:
Does this mean "start" and "stopandaccumulate" can be on different threads? Is a distribution with a single sample a suggested solution for someone who needs a cross-thread timespan?

Reply:
How about:

```
This metric can safely measure multiple concurrent timespans on different threads.
```

(Cross-thread timespans aren't really the intended use case, though since we use integer ids to identify the timespans, we can't stop you).  But I think this level of detail belongs in the reference of the timing distribution metric itself.